codegen/src/main/scala/com/soundcloud/twinagle/codegen/TwinagleServicePrinter.scala

```diff
@@ -106,35 +105,15 @@ final class TwinagleServicePrinter(service: ServiceDescriptor, implicits: Descri
 
   }
 
-  implicit final class ServiceExt(message: ServiceDescriptor) {
-
-    def sourcePath: Seq[Int] = Seq(FileDescriptorProto.SERVICE_FIELD_NUMBER, message.getIndex)
-
-    def comment: Option[String] = {
-      message.getFile
-        .findLocationByPath(sourcePath)
-        .map(t => t.getLeadingComments + t.getTrailingComments)
-        .map(Helper.escapeComment)
-        .filter(_.nonEmpty)
-    }
-  }
-
   private def generateServiceTrait(serviceDescriptor: ServiceDescriptor): String = {
-    val docLines: Seq[String] = serviceDescriptor.comment.map(_.split('\n').toSeq).getOrElse(Seq.empty)
-    val docString = if (docLines.nonEmpty) {
-      s"""
-         |/**
-         |${docLines.map("  * " + _).mkString("\n")}
-         |  */
-         """.stripMargin
-    } else {
-      ""
-    }
+    val comment = serviceDescriptor.comment
+    val docString: String = commentToDocString(comment)
     val serviceName = getServiceName(serviceDescriptor)
     val methods = serviceDescriptor.methods
     s"""
-       |${docString}trait $serviceName {
+       |$docString
```

Comment:
how does this look in the generated code? I'd imagine this would introduces some blank lines between the comment and the code

Reply:
yes it does, I can remove this